main.go

```diff
@@ -45,7 +45,7 @@ var (
 )
 
 const (
-	postgresURL = "host=postgresql.test-project.svc  port=5432 user=curator password=M5rBgWkN8LfjeyI8 dbname=curatordb sslmode=disable"
+	postgresURL = "host=<host-name>  port=<port-number> user=<username> password=<password> dbname=<database-name> sslmode=disable"
```

Comment:
It looks like we depend on this URL variable on line 87 so this would potentially break the report controller. Should we drop these changes and fix in a follow-up PR?

Reply:
But we can't expose the variables here in git repo. Whenever we will clone it we will apply variables before we run the operator.

Reply:
As discussed in Slack - dropping this change from current PR and we'll address this issue in a follow-up PR